cfg/const.go

```diff
@@ -227,4 +229,12 @@ var (
 		"Deposit",
 		"VotingPeriod",
 	}
+	ParamKeysTypeNumber = map[string]bool{
+		"VotingPeriod": true,
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File is not `gofmt`-ed with `-s` (gofmt)